src/components/scenes/Fio/FioCreateHandleScene.tsx

```diff
@@ -69,12 +69,18 @@ export const FioCreateHandleScene = ({ navigation, route }: Props) => {
   const inputRef = React.useRef<TextInput>(null)
   const mounted = React.useRef<boolean>(true)
 
+  // TODO: Give feedback to the user to indicate why the input was rejected
   const handleChangeFioHandle = useHandler((userInput: string) => {
-    // Clean the userInput:
-    userInput = userInput.replace(domainStr, '').trim()
+    // Dash '-' allowed,
+    // but cannot be the first...
+    if (userInput.indexOf('-') === 0) {
```

Comment:
Unnecessary change. Could have left the original line 77 and reduce regression potential.

Reply:
Line 77 was causing the bug where the user was unable to delete the first two characters via backspace